packages/augur-ui/src/modules/market-charts/components/depth/depth.tsx

```diff
@@ -45,9 +45,9 @@ const checkResize = memoize(
     MARGIN_OF_ERROR
 );
 function determineInitialZoom(props) {
-  const { orderBookKeys, marketMin, midPrice, marketMax } = props;
+  let { orderBookKeys, marketMin, midPrice, marketMax } = props;
```

Comment:
Instead of changing this to let, keep it const and remove the assignment of midPrice at all